geotrek/api/v2/filters.py

```diff
@@ -268,22 +266,22 @@ def filter_queryset(self, request, queryset, view):
             )
         return qs
 
-    def get_schema_fields(self, view):
+    def _get_schema_fields(self, view):
         return (
             Field(
                 name='near_trek', required=False, location='query', schema=coreschema.Integer(
                     title=_("Near trek"),
-                    description=_("Filter by a trek id. It will show only the touristics contents related to this trek.")
+                    description=_("Filter by a trek id. It will show only the contents related to this trek.")
                 )
             ), Field(
-                name='categories', required=False, location='query', schema=coreschema.Integer(
-                    title=_("Categories"),
-                    description=_("Filter by one or more category id, comma-separated.")
+                name='near_touristiccontent', required=False, location='query', schema=coreschema.Integer(
+                    title=_("Near touristic content"),
+                    description=_("Filter by a touristic content id. It will show only the contents related to this touristic content.")
                 )
             ), Field(
-                name='types', required=False, location='query', schema=coreschema.Integer(
-                    title=_("Types"),
-                    description=_("Filter by one or more types id, comma-separated. Logical OR for types in the same list, AND for types in different lists.")
+                name='near_touristicevent', required=False, location='query', schema=coreschema.Integer(
+                    title=_("Near touristic event"),
+                    description=_("Filter by a touristic event id. It will show only the vcontents related to this touristic event.")
```

Comment:
```suggestion
                    description=_("Filter by a touristic event id. It will show only the contents related to this touristic event.")
```